Cargo.toml

```diff
@@ -16,15 +21,15 @@ rpath = false
 [workspace.dependencies]
 axone-cognitarium = { path = "contracts/axone-cognitarium", features = [
   "library",
-] }
-axone-cognitarium-client = { path = "packages/axone-cognitarium-client" }
-axone-logic-bindings = { path = "packages/axone-logic-bindings" }
+], version = "5.0.0" }
```

Comment:
I understand the need of duplicating the version here, but it brings a new responsibility that is not managed here, this version must be in sync the new released versions. And sadly cargo doesn't come with CLI tools to achieved that 😢..

Currently at release each contracts & packages have their version updated by semantic release, this is configured in the [.releaserc.yml](https://github.com/axone-protocol/contracts/blob/f54a8f94ed4855e4163d2ef95cbe1558652e0935/.releaserc.yml#L31). However this replace plugin isn't suitable to update versions in the dependency list..

I think we could easily script this update using [toml-cli](https://github.com/gnprice/toml-cli), for example:
```bash
toml set Cargo.toml workspace.dependencies.axone-rdf.version 4.1.0
```

I think the changes made by `toml-cli` may need to be properly formatted to please the linter, and committed in the release commit.